metadata-ingestion/src/datahub/sdk/mlmodel.py

```diff
@@ -0,0 +1,328 @@
+            return version_props.version.versionTag
+        return None
+
+    @property
+    def aliases(self) -> Optional[List[str]]:
+        version_props = self._get_aspect(VersionPropertiesClass)
+        if version_props and version_props.aliases:
+            return [
+                alias.versionTag
+                for alias in version_props.aliases
+                if alias.versionTag is not None
+            ]
+        return None
+
+    def set_aliases(self, aliases: List[str]) -> None:
+        version_props = self._get_aspect(VersionPropertiesClass)
+        if version_props:
+            version_props.aliases = [
+                VersionTagClass(versionTag=alias) for alias in aliases
+            ]
+        else:
+            # If no version properties exist, we need to create one with a default version
+            version_set_urn = VersionSetUrn(
+                id=f"mlmodel_{self.urn.name}_versions", entity_type="mlModel"
+            )
+            self._set_aspect(
+                VersionPropertiesClass(
+                    version=VersionTagClass(versionTag="0.1.0"),  # Default version
+                    versionSet=str(version_set_urn),
+                    sortId="0000000.1.0",
+                    aliases=[VersionTagClass(versionTag=alias) for alias in aliases],
+                )
+            )
+
+    def add_aliases(self, aliases: List[str]) -> None:
+        version_props = self._get_aspect(VersionPropertiesClass)
+        if version_props:
+            if version_props.aliases is None:
+                version_props.aliases = []
+            # Check if alias already exists
+            for alias in aliases:
+                if not any(a.versionTag == alias for a in version_props.aliases):
+                    version_props.aliases.append(VersionTagClass(versionTag=alias))
+        else:
+            # If no version properties exist, we need to create one with the alias
+            self.set_aliases(aliases)
+
+    @property
+    def groups(self) -> Optional[List[str]]:
+        return self._ensure_model_props().groups
+
+    def set_group(self, group: str) -> None:
+        self._ensure_model_props().groups = [group]
+
+    def add_to_group(self, group_urn: str) -> None:
+        props = self._ensure_model_props()
+        if props.groups is None:
+            props.groups = []
+        props.groups.append(group_urn)
+
+    def remove_from_group(self, group_urn: str) -> None:
+        props = self._ensure_model_props()
+        if props.groups is not None:
+            props.groups = [group for group in props.groups if group != group_urn]
+
+    @property
+    def training_jobs(self) -> Optional[List[str]]:
+        return self._ensure_model_props().trainingJobs
+
+    def set_training_jobs(self, training_jobs: List[str]) -> None:
+        self._ensure_model_props().trainingJobs = training_jobs
+
+    def add_training_jobs(self, training_jobs: List[str]) -> None:
```

Comment:
will people really be using this method? or would a singular `add_training_job` be more useful?

these types of things are a judgement call, but don't just copy exactly what was done for other entity types